users/tests/conftest.py

```diff
@@ -29,6 +29,22 @@
 from users.models import Application, LoginMethod, OidcClientOptions, TunnistamoSession
 
 
+@pytest.fixture
+def use_translations(request):
+    """After the test, resets the currently active translation
+    to what it was before the test. Useful when a test potentially
+    changes the current active language."""
+    language_before_the_test = translation.get_language()
+
+    def reset_current_language():
+        if language_before_the_test:
+            translation.activate(language_before_the_test)
+        else:
+            translation.deactivate()
+
+    request.addfinalizer(reset_current_language)
```

Comment:
The `yield` style as recommended by the [pytest docs](https://docs.pytest.org/en/6.2.x/fixture.html#yield-fixtures-recommended) could be cleaner here?

Reply:
Yep. Changed.